src/crab_verifier.cpp

```
@@ -7,246 +7,192 @@
 }
 
-static string_invariant_map to_string_invariant_map(crab::invariant_table_t& inv_table) {
-    string_invariant_map res;
-    for (const auto& [label, inv] : inv_table) {
-        res.insert_or_assign(label, inv.to_set());
+std::set<std::string> Report::reachability_set() const {
+    std::set<std::string> result;
+    for (const auto& [label, warnings] : reachability) {
+        for (const auto& msg : warnings) {
+            result.insert(to_string(label) + ": " + msg);
+        }
```

cppcheck

[error] Syntax Error

(internalAstError)